notebooks/ch10/modeling_loss_functions.md

```diff
@@ -288,23 +375,23 @@ $$
 \end{aligned}
 $$
```

Comment:
I think that all of the thetas in this equation should have hats on them because they are the ideal theta!

![image](https://user-images.githubusercontent.com/39496826/43113447-4ecf5c80-8eaf-11e8-9a74-8b6200d35603.png)

Reply:
Hm... in the PDF attached on slack, Sam and Prof. Nolan left these alone, and I think it's because we haven't fitted it "yet", and by the last line it might be a little weird to suddenly slap on the hat. What do you think @SamLau95 ?